const.go

```diff
@@ -62,6 +62,13 @@ const (
 	RethinkDBBackend = "rethinkdb"
 
 	DefaultImportRole = "delegation"
+
+	// HealthCheckKeyManagement and HealthCheckSigner are the grpc service name
+	// for "KeyManagement" and "Signer" respectively which used for health check.
+	// An empty service name indicates the querying for overall status of the server.
+	HealthCheckKeyManagement = "grpc.health.v1.Health.KeyManagement"
+	HealthCheckSigner        = "grpc.health.v1.Health.Signer"
+	HealthCheckOverall       = ""
```

Comment:
sorry to make you change this, but I think it'd be better if this constant had a non-empty value defined.  Maybe something like `grpc.health.v1.Health.NotarySignerOverall`?